src/Input/Input.js

```diff
@@ -130,7 +130,7 @@ export default class Input extends Component {
     /**
      * The input value, required for a controlled component.
      */
-    value: PropTypes.string,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
I am fine with `any` since it lets me pass in a number without errors....but does it make sense to allow arrays and objects and functions for an input?  Or is it better that we just check something is sent in, and if your passing non scalar values that's on you?  I'd be glad to change this to any just let me know your thoughts on allowing non scalar values